src/functions-and-arrays.js

```diff
@@ -52,14 +121,46 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(c) {
```

Comment:
`function uniquifyArray(arr) {
  let newArr = []
  if (arr.length === 0) {
    return null
  } else {
    for (let i = 0; i < arr.length; i++) {
      if (newArr.indexOf(arr[i] === -1)) {
        newArr.push(arr[i])
      }
    }
    return newArr
  }
}`